public/docs/_examples/forms/dart/lib/hero.dart

```diff
@@ -0,0 +1,19 @@
+library hero_form.hero;
+
+class Hero {
+  num number;
+  String name;
+  String power;
+  String alterEgo;
+
+  Hero(this.number, this.name, this.power, {this.alterEgo});
+
+  Map toJson() {
```

Comment:
Nope! JSON.encode is not that fancy.

It should use the arrow, though 

```dart
Map toJson() => {'number: number ...}
```

Note that `this` is not needed